### ISSUE TITLE
Summary doesn't look good for special case all zero

### ISSUE THREAD

**Issue by xMartin** [xMartin/grouptabs#96]
![bildschirmfoto 2016-11-06 um 14 24 39](https://cloud.githubusercontent.com/assets/112532/20038057/e5ad318c-a42c-11e6-85dc-85643b530fc4.png)

**Comment by jancborchardt** [comment]
Yes, good point! Additionally, we can put in an element showing a success message like:

> 🎉
> **Hooray, you’re even!**
